packages/flutter_tools/test/commands.shard/permeable/create_test.dart

```diff
@@ -286,6 +286,106 @@ void main() {
     ),
   });
 
+  testUsingContext('creates a new app/skeleton project with pre-release flutter version constraint in pubspec', () async {
```

Comment:
Since CI runs against prerelease versions (main/beta), I added this test so we don't break that.